fastlane/lib/fastlane/actions/docs/frame_screenshots.md

```diff
@@ -215,6 +215,46 @@ The `keyword.strings` and `title.strings` are standard `.strings` file you alrea
 - These `.strings` files **MUST** be utf-8 (UTF-8) or utf-16 encoded (UTF-16 BE with BOM). They also must begin with an empty line. If you are having trouble see [issue #1740](https://github.com/fastlane/fastlane/issues/1740)
 - You **MUST** provide a background if you want titles. _frameit_ will not add the tiles if a background is not specified.
 
+### Screenshot orientation
+
+By default _frameit_ adds a frame to your screenshot based on an orientation you took it. For a portrait (vertical orientation) it is going to add portrait frame and for a landscape (horizontal orientation) - landscape left.
```

Comment:
What does "landscape left" actually mean here?

Reply:
According to Apple's [documentation](https://developer.apple.com/documentation/uikit/uiinterfaceorientation/landscapeleft):

> The device is in landscape mode, with the device held upright and the Home button on the left side.

So maybe it's worth to add this sentence? 

```
... for a landscape (horizontal orientation) - landscape left. The device is in landscape mode, with the device held upright and the Home button on the left side.
```

Not sure if it's okay to use the same sentence from Apple's documentations without including the source, so maybe better you can insert a link to the docs: `... - [landscape left](https://developer.apple.com/documentation/uikit/uiinterfaceorientation/landscapeleft)` ? 🤔

Reply:
Simple:
```suggestion
By default _frameit_ adds a frame to your screenshot based on an orientation you took it. For a portrait (vertical orientation) it is going to add portrait frame and for a landscape (horizontal orientation) - landscape left (= [Home button on the left side](https://developer.apple.com/documentation/uikit/uiinterfaceorientation/landscapeleft)).
```